src/server.js

```diff
@@ -53,9 +54,14 @@ module.exports = function() {
 			log.error("Path to SSL certificate is invalid. Stopping server...");
 			process.exit();
 		}
+		if (config.https.ca.length && !fs.existsSync(caPath)) {
```

Comment:
caPath is just config.https.ca with normalised "~" is it not? So what's the need to use config.https.ca? That's extra characters. If config.https.ca is "", then caPath will be "".